<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZeroLevelHistos</class>
 <widget class="QMainWindow" name="ZeroLevelHistos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,15">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Hist</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="2" column="4">
            <widget class="QCustomPlot" name="h8" native="true"/>
           </item>
           <item row="3" column="2">
            <widget class="QCustomPlot" name="h11" native="true"/>
           </item>
           <item row="3" column="0">
            <widget class="QCustomPlot" name="h9" native="true"/>
           </item>
           <item row="1" column="2">
            <widget class="QCustomPlot" name="h3" native="true"/>
           </item>
           <item row="1" column="1">
            <widget class="QCustomPlot" name="h2" native="true"/>
           </item>
           <item row="3" column="4">
            <widget class="QCustomPlot" name="h12" native="true"/>
           </item>
           <item row="1" column="0">
            <widget class="QCustomPlot" name="h1" native="true"/>
           </item>
           <item row="1" column="4">
            <widget class="QCustomPlot" name="h4" native="true"/>
           </item>
           <item row="2" column="0">
            <widget class="QCustomPlot" name="h5" native="true"/>
           </item>
           <item row="3" column="1">
            <widget class="QCustomPlot" name="h10" native="true"/>
           </item>
           <item row="2" column="2">
            <widget class="QCustomPlot" name="h7" native="true"/>
           </item>
           <item row="2" column="1">
            <widget class="QCustomPlot" name="h6" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Graph</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="2" column="4">
            <widget class="QCustomPlot" name="h8_2" native="true"/>
           </item>
           <item row="3" column="2">
            <widget class="QCustomPlot" name="h11_2" native="true"/>
           </item>
           <item row="3" column="0">
            <widget class="QCustomPlot" name="h9_2" native="true"/>
           </item>
           <item row="1" column="2">
            <widget class="QCustomPlot" name="h3_2" native="true"/>
           </item>
           <item row="1" column="1">
            <widget class="QCustomPlot" name="h2_2" native="true"/>
           </item>
           <item row="3" column="4">
            <widget class="QCustomPlot" name="h12_2" native="true"/>
           </item>
           <item row="1" column="0">
            <widget class="QCustomPlot" name="h1_2" native="true"/>
           </item>
           <item row="1" column="4">
            <widget class="QCustomPlot" name="h4_2" native="true"/>
           </item>
           <item row="2" column="0">
            <widget class="QCustomPlot" name="h5_2" native="true"/>
           </item>
           <item row="3" column="1">
            <widget class="QCustomPlot" name="h10_2" native="true"/>
           </item>
           <item row="2" column="2">
            <widget class="QCustomPlot" name="h7_2" native="true"/>
           </item>
           <item row="2" column="1">
            <widget class="QCustomPlot" name="h6_2" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Allan</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCustomPlot" name="allanPlot" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>PM:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string>A0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>N:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>131072</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>131072</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="startBtn">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="stopBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../src/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
